Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.17.5-alpine as builder
+FROM golang:1.18-alpine as builder
```

Comment:
Please use a more specific go version, e.g., 1.18.3 (as that's the last official supported version of Dynatrace oneagent https://www.dynatrace.com/support/help/technology-support/application-software/go/support/supported-go-versions )